script/script.js

```diff
@@ -34,3 +34,63 @@ lin.addEventListener('click', () => {
   mobileMen.style.display = 'none';
   mobileNav.style.display = 'block';
 });
+
+const projects = [{
+  title: 'Keeping track of hundreds of components',
+  image: 'assets/images/Snapshoot.svg',
+  description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea",
+  language1: 'Github',
+  language2: 'Codepen',
+  language3: 'JavaScript',
+  language4: 'Bootstrap',
+  language5: 'css',
+  language6: 'Terminal',
+  language7: 'Codekit',
+  language8: 'Ruby on Rails',
+}];
+
+const [project] = projects;
+
+const button = document.querySelectorAll('.works-button');
+const popup = document.querySelector('.popup-window');
+
+popup.innerHTML = `<div class="container">
+<button class="close-button"><i class="fa fa-times"></i></button>
+<img src=${project.image} alt="project-img" class="popup-img">
+<div class="header-container">
+    <h2 class="container-title">${project.title}</h2>
+    <div class="desktop butt">
+<button class='seelive'>See Live<span><img src="assets/images/live.svg" alt="live"></span></button>
+</div>
+<div class='source butt'>
+<button class='seesource'>See Source<span><img src="assets/images/source.png" alt="source"></span></button>
+</div>
+</div>
+<ul class="skillsb">
+<li class="mobile">${project.language1}</li>
+<li class="mobile">${project.language2}</li>
+<li class="mobile">${project.language3}</li>
+<li class="desktop">${project.language4}</li>
+<li class="desktop">${project.language5}</li>
+<li class="desktop">${project.language6}</li>
+<li class="desktop">${project.language7}</li>
+<li class="desktop">${project.language8}</li>
+</ul>
+<p class="description">${project.description}</p>
+<div class="buttons non">
+<button>See Live<span><img src="assets/images/live.svg" alt="github"></span></button>
+<button>See Source<span><img src="assets/images/source.png" alt="github"></span></button>
```

Comment:
_**On mobile **_
- [ ] **Required:** You have done a good job so far, however, I observed that you have not followed the Figma template in position some elements. Take a look below:

Your Design | Template
:---: | :---:
![image](https://user-images.githubusercontent.com/27709832/156457575-c7744a05-f70f-4c26-8b4b-77d0643461c6.png) | ![image](https://user-images.githubusercontent.com/27709832/156457245-c1cb4fd6-fad0-4ba1-832c-636564abd302.png)

- Kindly remove the white space at the top and add padding to the image.

Your Design | Template
:---: | :---:
![image](https://user-images.githubusercontent.com/27709832/156457930-3fe67c5d-024d-4549-a66f-5d63820096f9.png) | ![image](https://user-images.githubusercontent.com/27709832/156458151-6f181b6c-5172-41e9-9903-3782f26f86ed.png)

- Reposition the button and add padding to them appropriately 

**_On desktop_**

Your Design | Template
:---: | :---:
![image](https://user-images.githubusercontent.com/27709832/156459596-0939dbe9-71ac-4da0-b5f5-7e6303bb5fe3.png) | ![image](https://user-images.githubusercontent.com/27709832/156459032-2753f771-c46a-429d-8582-0ecd92151560.png)

- Kindly reposition the labeled elements on the popup.

Reply:
Addressed ✔